test/integration/npm/main.go

```diff
@@ -73,7 +74,7 @@ var (
 )
 
 func main() {
-	dp, err := dataplane.NewDataPlane(nodeName, common.NewIOShim(), dpCfg)
+	dp, err := dataplane.NewDataPlane(nodeName, common.NewIOShim(), dpCfg, nil)
```

Comment:
Any reason to keep the stop channel nil? Can we pass a legit channel?

Reply:
no reason. I can do that